### PR TITLE
Bugfixes for icons, rendering and unwanted background scanning

### PR DESCRIPTION
**Pull request by n9iels**
This PR fixes multiple bugs
- #126 - Due to outdated dependencies the icon font wouldn't load anymore
- During the display of the scanning results the camera and QR scanner was still active in the background.
- When a long list of data is claimed the validation result indication (aka. Approved/Denied and "Afgegeven door: ....") was not visible anymore
- With the new NLX Mock data the `name` property of the configurations `SOURCE_DATA_SELECTION` is pointing to a object. This causes render/crash issues.
- Fixing reported vulnerabilities by NPM and broken tests

## Test instructions
1. Start the application and confirm the home screen matches the screenshot in the README (thus icons are correctly loaded)
2. Walk through the validation process with the BSN numer `999994669` and confirm that for "Uittreksel Basis Registratie Persoonsgegevens" and "Uittreksel Basis Regsitratie Verplaatsplaatshistorie" all the information on the paper is also visible in the app.
3. Confirm that the layout for "Uittreksel Basis Regsitratie Verplaatsplaatshistorie" matches the screenshot below:
4. Open the logs of the application (http://localhost:19002/) and point the camera of the phone to the QR code while the validation result screen is open. There should be no log activity that indicates a QR code has been scanned

image

### CODE REVIEW

File: waardepapieren-service/configuration/waardepapieren-config-compose-travis.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   "PRODUCT_PURPOSE" : "Bewijs verblijfadres in woonplaats",
   "SOURCE_DATA_SELECTION" : [
     {"Burgerservicenummer (BSN)" : "_embedded.ingeschrevenpersonen[0].burgerservicenummer"},
-    {"Naam" : "_embedded.ingeschrevenpersonen[0].naam"},
+    {"Voorletters" : "_embedded.ingeschrevenpersonen[0].naam.voorletters"},
```
Comment (pimotte) [inline waardepapieren-service/configuration/waardepapieren-config-compose-travis.json:21]: Do we have a last name we can use here? (Same remark for the other configs) That's probably a better sample, data-wise. Or maybe have both Voorletters and last name.

Comment (n9iels) [reply waardepapieren-service/configuration/waardepapieren-config-compose-travis.json:21]: There is no last name remarkably enough :thinking: Currently  the name object contains:
```json
"naam": {
    "aanhef": "Geachte mevrouw 'S Streeveld-te Schaars",
    "aanschrijfwijze": "jonkvrouw Ż.Å.Đ. 'S Streeveld-te Schaars",
    "gebruikInLopendeTekst": "jonkvrouw 'S Streeveld-te Schaars",
    "aanduidingNaamgebruik": "eigen_partner",
    "voornamen": "Żáïŀëñøŕ Åłéèç Đëļŧå",
    "voorletters": "Ż.Å.Đ.",
    "geslachtsnaam": "Streeveld",
    "voorvoegsel": "'S"
}
```
We can do both voorletters and aanhef?

**Comment by n9iels** [comment]
The NPM audit is fixed, same for the cypress E2E tests :smile:

File: waardepapieren-service/configuration/waardepapieren-config-compose-travis.json (hunk 1 above)
Comment (pimotte) [reply waardepapieren-service/configuration/waardepapieren-config-compose-travis.json:21]: Geslachtsnaam is last name

Comment (n9iels) [reply waardepapieren-service/configuration/waardepapieren-config-compose-travis.json:21]: That was a unknown synonym to me. I added it to the test data of the configuration files

**Pull request merged**